internal/cmd/kafka/command_topic_consume.go

```diff
@@ -80,8 +80,11 @@ func (c *command) consume(cmd *cobra.Command, args []string) error {
 	if err != nil {
 		return err
 	}
+	var commitGroupOffset bool
 	if !cmd.Flags().Changed("group") {
 		group = fmt.Sprintf("confluent_cli_consumer_%s", uuid.New())
+	} else {
+		commitGroupOffset = true
```

Comment:
Potential edge case: What if someone uses a default group ID the first time they call `confluent kafka topic consume`, and then they run `confluent kafka topic consume --group <default-group-id-from-the-first-run>`?

Reply:
In other words, maybe we should always commit the group offset?

Reply:
if it's set to true by default then i think yes, we don't need to change that. i should rename this var to be `consume from committed offset instead of rebalanceCallback`